app/src/main/res/values/styles.xml

```diff
@@ -345,6 +345,21 @@
         <item name="sv_tickRotation">false</item>
     </style>
 
+    <style name="AppTheme.MK" parent="AppTheme">
```

Comment:
Create a new XML element in this file to declare your new theme. The item in `ThemeList` is pointing to this.